odoo/addons/base/models/res_company.py

```diff
@@ -190,8 +190,8 @@ def _company_default_get(self, object=False, field=False):
         _logger.warning("The method '_company_default_get' on res.company is deprecated and shouldn't be used anymore")
         return self.env.company
 
-    # deprecated, use clear_caches() instead
     def cache_restart(self):
+        warnings.warn("Since 16.0, deprecated method, use `clear_caches` instead", DeprecationWarning, 2)
```

Comment:
That's a lie! You're deprecating it in 17.0! It's almost as bad as when people don't even put a deprecation version in the PR!

![other](https://user-images.githubusercontent.com/7571158/197196247-32fb4206-ff6c-4b86-a333-3c99555a0412.png)

Reply:
Oupsi! I fixed it :+1: 

> (but seriously I was thinking it might be a good idea to require version numbers in deprecation messages via a linter, it's annoying having to find out what branch a commit is on every time)

(Agree!)